Source/RapyutaSimulationPlugins/Public/Robots/RRRobotROS2Interface.h

```diff
@@ -116,8 +116,10 @@ class RAPYUTASIMULATIONPLUGINS_API URRRobotROS2Interface : public UObject
     /**
      * @brief Setup ROS Params, overridable by child classes to config custom ROS2 Interface's params
      */
-    UFUNCTION()
-    virtual void SetupROSParams();
+    UFUNCTION(BlueprintNativeEvent, BlueprintCallable)
+    void SetupROSParams();

```

Comment:
This is a virtual function, meaning it is currently re-implemented by various other classes, so it should be kept.
For BP implementation support, it'd be better to create another one calling this function.